modules/dxp/apps/portal-workflow/portal-workflow-kaleo-forms-web/src/main/resources/META-INF/resources/admin/record_action.jsp

```diff
@@ -61,4 +61,10 @@ DDLRecordVersion ddlRecordVersion = ddlRecord.getLatestRecordVersion();
 			url="<%= deleteDDLRecordURL %>"
 		/>
 	</c:if>
+
+	<liferay-ui:icon
```

Comment:
Hey @MarinhoFeliphe,

it seems weird to me. We have a class that is a taglib but we don't have a tld file and by consequence, we are not using it as such. Now, we have a *Util/*Helper class with the taglib name.

Thx